### DISCUSSION TITLE
YouTube embed start times

### DISCUSSION THREAD

**Discussion by addyosmani** [fancyapps/ui#195] [Q&A] [answered]
I am using Fancybox 4 (great work) and have been trying to get YouTube embeds working with the `start` or `t` parameters to begin at a particular time. Playing from a specific time point seems to work just fine in Fancybox [3.x](https://codepen.io/addyosmani/pen/ExwwbWp?editors=1010) but isn't with 4.x. Does anyone have tips on getting this working?

Examples of the kinds of URLs I'm referring to:
```
https://www.youtube.com/watch?v=dZRqB0JLizw&t=40s
https://youtu.be/dZRqB0JLizw?t=20
```

**Comment by fancyapps** [comment, marked as answer]
Hi,

Thanks for reporting. It should work now (after upgrading to the latest) - https://fancyapps.com/playground/1z5

**Reply by addyosmani** [reply]
Amazing! Thanks so much, @fancyapps! 😄 I appreciate the fix!
